components/forms/w-textarea.vue

```diff
@@ -1,22 +1,33 @@
 <template>
   <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation }">
-    <div class="input input--textarea mb-0">
-      <textarea v-bind="{ ...$attrs, class: '' }" v-model="model" :id="id" @blur="triggerValidation" />
+    <div :class="wrapperClass">
```

Comment:
`wrapperClass` adds only `relative` class, so doesn't include `mb-0`. Is that correct? 

Haven't found where `input--textarea` was styled in the linked css repo - was that why it was removed?

Reply:
I wonder if we even need this `div` and `relative` here. We don't have wrapper like that in `react` for example.

Reply:
Maybe it makes no difference in textarea, but I'm now wondering - was mb-0 supposed to be applied to all input components by default since it's part of `input.default`?

Reply:
And it is in select here in vue and react

Reply:
Sounds good!

Reply:
I love the questions as it also helps to think beyond my own 🧠 and see other scenarios. Shall we remove the wrapper div you think?

Reply:
If I understand correctly, react's textarea component allowed additional classes to be set on the wrapping div. Here it wasn't applied, but it was on `w-input` in the form of `inputWrapperClass` prop. Maybe instead of removing the div we should consider adding support for those additional classes? Then maybe that `wrapperClass` could also be removed? At least for the sake of consistency with whatever was introduced in React. I'm really unsure how setting `relative` on the `textarea` or `input` element instead of a wrapping div makes a difference.

Reply:
I guess we need `relative` if we have some icons(e.g. chevron) involved. For textarea it is not a common pattern, more for textfields and selects. `wrapperClass` in textarea wasn't here from the beginning. It is something I introduced in this PR.